pybliometrics/scopus/utils/startup.py

```diff
@@ -48,18 +49,27 @@ def check_sections(config: Type[ConfigParser]) -> None:
         if not config.has_section(section):
             raise NoSectionError(section)
 
+
 def check_default_paths(config: Type[ConfigParser], config_path: Type[Path]) -> None:
     """Auxiliary function to check if default cache paths exist.
-    If not, the paths are writen in the config"""
+    If not, the paths are writen in the config."""
```

Comment:
For multi-line docstrings, the tripe quote ends on a newline ([PEP 257](https://peps.python.org/pep-0257/#multi-line-docstrings)).